osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -14,11 +15,13 @@ public class GlobalActionContainer : DatabasedKeyBindingContainer<GlobalAction>,
     {
         private readonly Drawable handler;
 
-        public GlobalActionContainer(OsuGameBase game)
+        public GlobalActionContainer(OsuGameBase game, bool nested = false)
             : base(matchingMode: KeyCombinationMatchingMode.Modifiers)
         {
             if (game is IKeyBindingHandler<GlobalAction>)
                 handler = game;
+
+            GetInputQueue = () => base.KeyBindingInputQueue.ToArray();
```

Comment:
What is the copy here for? Previous code doesn't seem to have needed one.

Reply:
As above, unneeded.